### ISSUE TITLE
Unable to add show from TheTVDB - FBI: Most Wanted

### ISSUE THREAD

**Issue by Mini-Myself** [pymedusa/Medusa#8745]
**Describe the bug**
When trying to add the show "FBI: Most Wanted", it shows up when searching for it, but when hitting add show it doesn't add it.

**To Reproduce**
1. Shows > Add Shows
2. Add New Show
3. Search for "FBI: Most Wanted"
4. Click Add Show

**Expected behavior**
That the show gets added.

**Screenshots**

**Medusa (please complete the following information):**
 - OS: Windows 10 x64
 - Branch: master
 - Commit: f39927ea6f0b3d18856dce32d86af43d57713720
 - Python version: 3.7.9
 - Database version: 44.17

**Debug logs (at least 50 lines):**
<details>

```
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] Getting all show data for 94372
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] 364088: Loading show info from IMDb with ID: tt9742936
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] Retrieving show info from IMDb
2020-11-17 01:00:51 INFO	SHOWQUEUE-ADD :: [f39927e] Setting all previously aired episodes to the specified status: Skipped
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] TVDBv2 364088: Setting location: D:\Series\FBI Most Wanted
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] Transaction with 4 queries executed
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] Trying other indexer: TMDB get_id_by_external
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] User-Agent: Medusa/0.4.5 (Windows; 10; a42ef012-2797-11eb-9776-107b4449195d)
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] GET URL: http://api.tvmaze.com/shows/40695 [Status: 200]
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] User-Agent: Medusa/0.4.5 (Windows; 10; a42ef012-2797-11eb-9776-107b4449195d)
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] GET URL: http://api.tvmaze.com/shows/40695 [Status: 200]
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] User-Agent: Medusa/0.4.5 (Windows; 10; a42ef012-2797-11eb-9776-107b4449195d)
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] GET URL: http://api.tvmaze.com/lookup/shows?thetvdb=364088 [Status: 301]
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] Trying other indexer: TVmaze get_id_by_external
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] 364088: Loading show info from TVDBv2
2020-11-17 01:00:51 INFO	SHOWQUEUE-ADD :: [f39927e] 364088: Unable to find the show in the database
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] 364088: Loading show info from database
2020-11-17 01:00:51 DEBUG	SHOWQUEUE-ADD :: [f39927e] Trying other indexer: TMDB get_id_by_external
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] User-Agent: Medusa/0.4.5 (Windows; 10; a42ef012-2797-11eb-9776-107b4449195d)
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] GET URL: http://api.tvmaze.com/shows/40695 [Status: 200]
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] User-Agent: Medusa/0.4.5 (Windows; 10; a42ef012-2797-11eb-9776-107b4449195d)
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] GET URL: http://api.tvmaze.com/shows/40695 [Status: 200]
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] User-Agent: Medusa/0.4.5 (Windows; 10; a42ef012-2797-11eb-9776-107b4449195d)
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] GET URL: http://api.tvmaze.com/lookup/shows?imdb=tt9742936 [Status: 301]
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] Trying other indexer: TVmaze get_id_by_external
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] User-Agent: Medusa/0.4.5 (Windows; 10; a42ef012-2797-11eb-9776-107b4449195d)
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] GET URL: https://api.thetvdb.com/series/364088/episodes/query?page=1 [Status: 200]
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] Getting all episodes of 364088
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] User-Agent: Medusa/0.4.5 (Windows; 10; a42ef012-2797-11eb-9776-107b4449195d)
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] GET URL: https://api.thetvdb.com/series/364088 [Status: 200]
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] Getting all show data for 364088
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] Getting all series data for 364088
2020-11-17 01:00:50 DEBUG	SHOWQUEUE-ADD :: [f39927e] Configured language en override show language of en
2020-11-17 01:00:50 INFO	SHOWQUEUE-ADD :: [f39927e] TVDBv2: {'language': 'en', 'use_zip': True, 'session': <medusa.session.core.MedusaSession object at 0x000001A0B0104288>, 'cache': 'C:\\Medusa\\Data\\cache\\indexers\\TVDBv2'}
2020-11-17 01:00:50 INFO	SHOWQUEUE-ADD :: [f39927e] Starting to add show by show_dir: D:\Series\FBI Most Wanted
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] Using custom UI: <class 'medusa.classes.AllShowsListUI'>
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] Searching for show: fbi: most wanted
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] Searching for show with search term(s): ['fbi: most wanted'] on indexer: tmdb
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] Using custom UI: <class 'medusa.classes.AllShowsListUI'>
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] User-Agent: Medusa/0.4.5 (Windows; 10; a42ef012-2797-11eb-9776-107b4449195d)
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] GET URL: http://api.tvmaze.com/search/shows?q=fbi%3A%20most%20wanted [Status: 200]
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] Searching for show fbi: most wanted
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] Searching for show with search term(s): ['fbi: most wanted'] on indexer: tvmaze
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] Using custom UI: <class 'medusa.classes.AllShowsListUI'>
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] User-Agent: Medusa/0.4.5 (Windows; 10; a42ef012-2797-11eb-9776-107b4449195d)
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] GET URL: https://api.thetvdb.com/search/series?name=fbi%3A+most+wanted [Status: 200]
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] Searching for show: fbi: most wanted
2020-11-17 01:00:40 DEBUG	APIv2-Thread-32 :: [f39927e] Searching for show with search term(s): ['fbi: most wanted'] on indexer: tvdb
```
</details>

**Additional context**
-

**Comment by p0psicles** [comment]
you sure the log stops there?
then it looks like your having issues connecting to some of the api's?

**Comment by Mini-Myself** [comment]
I can access their website just fine. Is there some way I can check the connection with the API or something?
Medusa has network access, everything else works fine. Not sure how to fix this if that is the cause.

**Comment by Mini-Myself** [comment]
I managed to get the error! Not sure why, but it only shows up when running Medusa in a command prompt. The error is not placed in the log file.

```
Exception in thread SHOWQUEUE-ADD:
Traceback (most recent call last):
  File "C:\Python37\lib\threading.py", line 926, in _bootstrap_inner
    self.run()
  File "C:\Medusa\Medusa\medusa\queues\show_queue.py", line 559, in run
    self.show.load_imdb_info()
  File "C:\Medusa\Medusa\medusa\tv\series.py", line 1630, in load_imdb_info
    imdb_info = imdb_api.get_title(self.imdb_id)
  File "C:\Medusa\Medusa\ext\imdbpie\imdbpie.py", line 81, in get_title
    '/title/{0}/auxiliary'.format(imdb_id)
  File "C:\Medusa\Medusa\ext\imdbpie\imdbpie.py", line 299, in _get_resource
    return self._get(url=url)['resource']
  File "C:\Medusa\Medusa\ext\imdbpie\imdbpie.py", line 307, in _get
    headers.update(self.get_auth_headers(full_url))
  File "C:\Medusa\Medusa\ext\imdbpie\auth.py", line 112, in get_auth_headers
    creds, soon_expires = self._creds_soon_expiring()
  File "C:\Medusa\Medusa\ext\imdbpie\auth.py", line 97, in _creds_soon_expiring
    creds = self._get_creds()
  File "C:\Medusa\Medusa\ext\imdbpie\auth.py", line 85, in _get_creds
    return cache.get(self._CREDS_STORAGE_KEY)
  File "C:\Medusa\Medusa\ext\diskcache\core.py", line 907, in get
    value = self._disk.fetch(mode, filename, db_value, read)
  File "C:\Medusa\Medusa\ext\diskcache\core.py", line 279, in fetch
    return pickle.load(BytesIO(value))
ValueError: unsupported pickle protocol: 5

**Comment by p0psicles** [comment]
Maybe the easiest fix for this is upgrading to python 3.8. Can you do that?

**Comment by Mini-Myself** [comment]
I upgraded to 3.8, this fixes the issue.
